File: Easy/palindrome_number.php
<?php

class Solution
{

    /**
     * @param Integer $x
     * @return Boolean
     */
    function isPalindrome($x)
    {
        $strX = strval($x);
        return strrev($strX) === $strX;
    }
}

$solution = new Solution();

print_r($solution->isPalindrome(121)); //true
print_r($solution->isPalindrome(-121)); //false
print_r($solution->isPalindrome(10)); //false

File: Easy/palindrome_number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        strX = str(x)
        return strX == strX[::-1]
    
print(Solution().isPalindrome(121)) #true
print(Solution().isPalindrome(-121)) #false
print(Solution().isPalindrome(10)) #false